client/stylesheets/_modal.scss

```diff
@@ -233,14 +233,55 @@
 	END END END END END END
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
 
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+	History Details Modal
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+
+/*~~~~ Button ~~~~*/
 
+/*~~ Button End ~~*/
+
+#profileHistoryFeedback {
+	width: 100% !important;
```

Comment:
!important should not be used<br>Line should be indented with spaces, not tabs<br>Properties should be ordered background-color, border-radius, color, padding, text-align, text-decoration, width